custom/icds_reports/static/icds_reports/js/spec/enrolled-children.directive.spec.js

```diff
@@ -14,24 +14,7 @@ describe('Enrolled Children Directive', function () {
     pageData.registerUrl('icds_locations', 'icds_locations');
 
     beforeEach(module('icdsApp', function ($provide) {
-        $provide.constant("genders", [
-            {id: '', name: 'All'},
-            {id: 'M', name: 'Male'},
-            {id: 'F', name: 'Female'},
-        ]);
-        $provide.constant('ages', [
-            {id: '', name: 'All'},
-            {id: '6', name: '0-6 months'},
-            {id: '12', name: '6-12 months'},
-            {id: '24', name: '12-24 months'},
-            {id: '36', name: '24-36 months'},
-            {id: '48', name: '36-48 months'},
-            {id: '60', name: '48-60 months'},
-            {id: '72', name: '60-72 months'},
-        ]);
-        $provide.constant("userLocationId", null);
-        $provide.constant("haveAccessToAllLocations", false);
-        $provide.constant("isAlertActive", false);
+        utils.provideDefaultConstants($provide, true, true);
```

Comment:
Not blocking: I wish js supported named params; the calls with multiple booleans are hard to read and easy to mess up. What would you think of making `provideDefaultConstants` consume a parameters object instead? Like:
```
module.provideDefaultConstants = function ($provide, options) {
   hqImport("hqwebapp/js/assert_properties").assert(options, [], ['includeGenders', 'includeAges', 'overrides']);
   ...
   if (options.includeGenders) {
      ...
```

Reply:
ha: https://github.com/dimagi/commcare-hq/pull/26140#discussion_r355191976

I agree that would be better. I got hung up with providing both `options` and `overrides` though I guess I could make `overrides` a property of options. 

I'll make a backlog item for this and hopefully get to it soonish.

Reply:
Haha, glad we're aligned. Doing `overrides` as a separate param and just using options for the booleans seems fine, too.

Reply:
https://github.com/dimagi/commcare-hq/pull/26484